Client/Frontend/Home/HomePanelTableViewController.swift

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Fix indentation. I really recommend using [this](https://www.dropbox.com/s/9l38op33lzhx408/Templates.zip?dl=0) (extract it to <code>~/Library/Developers/Xcode</code>) to make this annoyance go away.